_i18n/es.yml

```diff
@@ -0,0 +1,54 @@
+# 1.0.0
+langs:
+  es: Español
+home:
+  title: |
+    <span>Poder</span> para los trabajadores en la industria de la tecnología
+    summary: Guiados por nuestra visión para una industria de la teconogía mas equitativa e inclusiva, TWC se organiza para construir un poder para los trabajadores mediante la educación y autoorganizacion
```

Comment:
Should this line be less indented? (I cannot test it at the moment)

Reply:
me too but I didn't change the identation. So I think it should work